src/Aggregation/Cardinality.php

```diff
@@ -9,6 +9,7 @@
  */
 class Cardinality extends AbstractSimpleAggregation
 {
+    use MissingTrait;
```

Comment:
CS, but I would in favor to add a line break to clearly separate from following constant declaration.